crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -63,9 +64,17 @@ fn initialize<B: Board>(mut call: SchedulerCall<B, api::initialize::Sig>) {
     let result = try {
         let context = match convert_hash_algorithm::<B>(*algorithm)?? {
             #[cfg(feature = "board-api-crypto-sha256")]
-            Algorithm::Sha256 => HashContext::Sha256(board::crypto::Sha256::<B>::default()),
+            Algorithm::Sha256 => {
+                let default = board::crypto::Sha256::<B>::default();
+                default.last_error()?;
+                HashContext::Sha256(default)
+            }
```

Comment:
@ia0 You mentioned we could use `Result::map` for this. Could you explain more? Thanks.